chain-core/src/init/network.rs

```diff
@@ -71,8 +71,8 @@ pub fn get_bech32_human_part() -> &'static str {
 
 mod chosen_network {
     use super::*;
-    pub static mut NETWORK: Network = Network::Devnet;
-    pub static mut NETWORK_ID: u8 = 0 as u8;
+    pub static mut NETWORK: Network = Network::Testnet;
+    pub static mut NETWORK_ID: u8 = 0x42 as u8;
```

Comment:
Is there any specific reason to change these defaults?

Reply:
no specific reason
devnet is also good.
for devnet
how about 0xab?
@tomtau

Reply:
just keep it 0, it'll mean that it wasn't initialised

Reply:
for devnet, anyone can choose any number

Reply:
ok, i will change back